src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -731,6 +732,16 @@ public class SimpleEvents {
 				)
 				.since("2.7");
 
+		if (Skript.isRunningMinecraft(1, 14))
+			Skript.registerEvent("Player Take Lectern", SimpleEvent.class, PlayerTakeLecternBookEvent.class, "[player] tak(e|ing [a]) (lectern book|book from [a] lectern)")
```

Comment:
I'm a bit concerned about the name being "take lectern" when it's actually taking a book, not taking a whole lectern.

Reply:
It's taken from the name of the event, to keep familiarity to the event.

Reply:
> It's taken from the name of the event, to keep familiarity to the event.

It needs to include the `book` part too. The event is Taking a Lectern Book, not Taking a Lectern.